Chinese/Xinyang/MyLoveStory.md

```diff
@@ -0,0 +1,2 @@
+至尊宝：姑娘，我发表人生感悟你为何总是喋喋不休？月光什么光什么宝什么盒，我根本听都
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.